pkg/apis/hco/v1beta1/hyperconverged_types_test.go

```diff
@@ -358,6 +358,62 @@ var _ = Describe("HyperconvergedTypes", func() {
 			})
 		})
 
+		Context("Test IsGPUAssignmentEnabled", func() {
+			It("Should return false if HyperConvergedFeatureGates is nil", func() {
+				var fgs *HyperConvergedFeatureGates = nil
+				Expect(fgs.IsGPUAssignmentEnabled()).To(BeFalse())
+			})
+
+			It("Should return false if IsGPUAssignmentEnabled does not exist", func() {
+				fgs := &HyperConvergedFeatureGates{}
+				Expect(fgs.IsGPUAssignmentEnabled()).To(BeFalse())
+			})
+
+			It("Should return false if IsGPUAssignmentEnabled is false", func() {
+				disabled := false
+				fgs := &HyperConvergedFeatureGates{
+					GPU: &disabled,
+				}
+				Expect(fgs.IsGPUAssignmentEnabled()).To(BeFalse())
+			})
+
+			It("Should return false if IsGPUAssignmentEnabled is true", func() {
```

Comment:
"should return **true** if IsGPUAssignmentEnabled is true"
@vladikr

Reply:
uhh, yeah. the title is wrong. I'll get it fixed in my next PR. Thanks.